qutebrowser/config/configdata.yml

```diff
@@ -1171,6 +1171,25 @@ statusbar.position:
   default: bottom
   desc: Position of the status bar.
 
+statusbar.widgets:
```

Comment:
Per https://github.com/qutebrowser/qutebrowser/issues/3269#issuecomment-343818412, @The-Compiler seemed to be leaing towards an array of widgets, but I like the idea of being able to put arbitrary text in the statusbar. I think it's fairly simple to convert between the two, so this shouldn't be a big issue. There is a bit of concern about how to escape the `{format}` fields though, so we might want to think about that a bit. (There's nothing actionable here, I'm just rambling)